### ISSUE TITLE
speed up the calculation when using function eot()

### ISSUE THREAD

**Issue by Mew-YL** [environmentalinformatics-marburg/remote#6]
For downscaling purpose, I have about 4GB MODIS NDVI data and  try to use the eot to downscale GIMMS NDVI data, but the calculation speed is too slow with such large dataset( only used 400MB MODIS NDVI data, I already waited for 5 hours ) and I wonder is there any possible ways to accelerate the calculation?

**Comment by tim-salabim** [comment]
I don't think there's an easy way to speed things up. `eot()` is already programmed in C++

**Comment by Mew-YL** [comment]
Can eot() run in parallel processing? just as mentioned in https://strimas.com/post/processing-large-rasters-in-r/

**Comment by tim-salabim** [comment]
No, parallel processing is not implemented. It has been a long time since we worked on this, but IIRC parallel processing is not feasible, because every pixel in x needs to be regressed against each pixel in y.
